src/vsr/replica.zig

```diff
@@ -269,6 +270,8 @@ pub fn ReplicaType(
         grid_repair_tables: IOPS(RepairTable, constants.grid_missing_tables_max) = .{},
         grid_repair_writes: IOPS(BlockWrite, constants.grid_repair_writes_max) = .{},
         grid_repair_write_blocks: [constants.grid_repair_writes_max]BlockPtr,
+        grid_scrubber: GridScrubber,
+        grid_scrubber_reads: IOPS(GridScrubber.Read, constants.grid_scrubber_reads_max) = .{},
```

Comment:
It it feels a bit odd that Replica owns these reads, rather than the scrubber directly.

Reply:
:+1: I'll change it so that `read_next()` returns a bool (whether a new read was started), and change `read_reclaim` to `read_fault() -> ?BlockReference`.